src/app/shared/util/chart.util.ts

```diff
@@ -145,3 +150,159 @@ export const minY = (chartType: ChartType): number => {
     return 0;
   }
 };
+
+export const aggregateTotalDownloads = (episodeTotals: EpisodeTotals[]) => {
+  return episodeTotals && episodeTotals.reduce((accTotal, et: EpisodeTotals) => {
```

Comment:
I did not know you could do this! Chain `&&` and get the final value if all evaluate to `true`. Nifty.

Reply:
These kind of null safety checks are definitely something that I've seen our Angular apps error on, so I try to be careful about them.